apps/docs/docs/tutorials/api-sdk/groups.md

```diff
@@ -0,0 +1,327 @@
+    name: "Manual Group",
+    description: "This is a description",
+    treeDepth: 16,
+    fingerprintDuration: 3600
+}
+
+const group = apiSdk.createGroup(groupCreationDetails, apiKey)
+```
+
+### Credential group
+
+Create a new credential group using the API SDK.
+
+```ts
+const apiKey = "your-api-key"
+
+const credentials = {
+    id: "BLOCKCHAIN_BALANCE",
+    criteria: {
+        minBalance: "10",
+        network: "Sepolia"
+    }
+}
+
+const groupCreationDetails = {
+    name: "Credential Group",
+    description: "This is a description",
+    treeDepth: 16,
+    fingerprintDuration: 3600,
+    credentials
+}
+
+const credentialGroup = apiSdk.createGroup(groupCreationDetails, apiKey)
+```
+### Multiple credentials group
+
+Create a multiple-credentials group using the API SDK
+
+```ts
+const apiKey = "your-api-key"
+
+const credentials = {
+    credentials: [
+        {
+            id: "BLOCKCHAIN_TRANSACTIONS",
+            criteria: {
+                minTransactions: 10,
+                network: "Sepolia"
+            }
+        },
+        {
+            id: "BLOCKCHAIN_BALANCE",
+            criteria: {
+                minBalance: "5",
+                network: "Sepolia"
+            }
+        }
+    ],
+    expression: ["", "and", ""]
+}
+
+const groupCreationDetails = {
+    name: "Multiple Credentials Group",
+    description: "This is a description",
+    treeDepth: 16,
+    fingerprintDuration: 3600,
+    credentials
+}
+
+const credentialGroup = apiSdk.createGroup(groupCreationDetails, apiKey)
+```
+More details on the credentials can be found at the [Credentials library](https://github.com/bandada-infra/bandada/tree/main/libs/credentials).
+
+## Add members to a group
+
+### Manual group
+
+You can add users directly to a manual group by using the API SDK.
+
+```ts
+const groupId = "your-group-id"
+const memberId = "member-id-1"
+const apiKey = "your-api-key"
+
+await apiSdk.addMemberByApiKey(
+    groupId,
+    memberId,
+    apiKey
+)
+```
+
+### Add members using an invite code
+
+Using the API SDK, you can invite users to join a manual group by sharing an invite code.
+
+#### Create invite
+
+Create a new group invite.
+
+```ts
+const groupId = "your-group-id"
+const apiKey = "your-api-key"
+
+const invite = await apiSdk.createInvite(groupId, apiKey)
+```
+
+You can wrap the invite code into a custom URL for your app, where it will handle the logic for adding a member to the group by invite code.
+
+Here is an example of the custom URL structure.
+
+```
+https://<custom-domain>?inviteCode=<invite-code>
+```
+
+#### Get invite
+
+Returns a specific invite along with the group details associated with the invite.
+
+```ts
+const inviteCode = "INVITECODE"
+const apiKey = "your-api-key"
+
+const invite = await apiSdk.getInvite(inviteCode)
+```
+
+#### Add member using an invite code
+
+Adds a member to a group using an invite code.
+
+```ts
+const groupId = "your-group-id"
+const apiKey = "member-id-1"
```

Comment:
```suggestion
const memberId = "member-id-1"
```

Reply:
Ah missed it, thanks for catching this error!